src/closure_externs.js

```diff
@@ -78,3 +78,168 @@ var PromiseConstructorLike;
 /** @typedef {function(new:symbol)} */
 var SymbolConstructor;
 
+/**
+ * @constructor
+ * @param {(undefined|!AnimationEffectReadOnly)=} effect
+ * @param {(undefined|!AnimationTimeline)=} timeline
+ */
+var Animation = function(effect, timeline) {}
```

Comment:
I'm not sure this belongs into tsickle. The other externs we have here are mostly for fundamental platform builtins like `Symbol` and friends (and HTMLTableCell etc maybe shouldn't be here).

Web animations seems pretty orthogonal, and also just a ton of code. Can you consider what it'd take to find a better home for this, e.g. in a different internal location that you'd add to the default set of included externs?